code/game/machinery/doors/airlock.dm

```diff
@@ -1272,6 +1274,12 @@ About the new airlock wires panel:
 	// </worry>
 	return ..()
 
+/obj/machinery/door/airlock/Uncross(atom/movable/mover)
+	if(density)
+		to_chat(mover, "You are pinned inside the closed airlock, you can't move!")
```

Comment:
Comma splices hurt me

Reply:
Won't this spam the doorstuckee whenever outside factors try to move them?